pyproject.toml

```diff
@@ -28,7 +28,7 @@ dependencies = [
   "Jinja2>=2.11.3",
   "MarkupSafe==2.0.1",
   "PyYAML~=6.0",
-  "click>=7.1.2, <8.1.4",
+  "click>=7.1.2",
```

Comment:
Oh cool, I wasn't sure if we could relax that prod version pin.

Reply:
in hindsight, I probably could've just did that version pin in the dev-dependency, rather than the prod one 😬